lib-model/lib-model-consume/build.gradle

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (C) 2017. Uber Technologies
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+plugins {
+    id "java-library"
+    id "nullaway.java-test-conventions"
+}
+
+jar{
+    manifest {
+        attributes('Main-Class':'com.uber.nullaway.libmodel.LibModelInfoExtractor')
+    }
+    from {
+        configurations.runtimeClasspath.collect { it.isDirectory() ? it : zipTree(it) }
+    }
+}
+
+dependencies {
+    testImplementation deps.test.junit4
+    testImplementation(deps.build.errorProneTestHelpers) {
+        exclude group: "junit", module: "junit"
+    }
+    testImplementation project(":nullaway")
+    testImplementation project(":lib-model:lib-model-test")
+    implementation deps.build.guava
+    implementation deps.build.javaparser
+    compileOnly deps.apt.autoService
+    annotationProcessor deps.apt.autoService
```

Comment:
Why do we need autoservice?

Reply:
I moved around some code while creating lib-model-test so it looks like I didn't remove this, it is not needed, I will remove it